salesforce-marketing-cloud/3.1/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,417 @@
+. Click on the trigger card.
+. Search for the connector.
+. Select the connector name.
+. Select the *On modified object* or *On new object* operation to use it as a source.
++
+Alternatively, you can also use the *HTTP Listener* or *Scheduler* component as a source.
++
+. Click *Click here to set it up*.
+. To create a global element for the connector, enter values for the Basic Authentication (Username Password) or OAuth Username Password authentication types:
++
+image::salesforce-mktg-3-dc-choose-global-type.png[Design Center global element]
++
+** For Basic Authentication (Username Password):
+* *Username* - Username used to initialize the session
+* *Password* - Password used to authenticate the
+* *Service URL* - URL of the service on Salesforce Marketing Cloud to be accessed from the connector
++
+** For OAuth Client Credentials authentication:
+* *Service Url* - URL of the Salesforce Marketing Cloud service that the connector will access
+*Client id* - Client ID for the installed package
+* *Client secret* - Client secret for the installed package
+* *Token url* - URL of the endpoint that provides the access tokens
++
+. Select the plus sign to add a component.
+. Select the connector as a component.
+. Configure these fields:
++
+* *Create* operation:
+** **Object Type** - Type of API object to create
+** *Api Objects* - Array of one or more API objects
++
+image::salesforce-mktg-3-dc-create.png[Create fields]
++
+* *Delete* operation:
++
+** *Object Type* - Type of API object to delete
+** *Api Objects* - Array of one or more API objects
++
+image::salesforce-mktg-3-dc-delete.png[Delete fields]
++
+* *Update* operation:
++
+** *Object Type* - Type of API object to update
+** *Api Objects* - Array of one or more API objects
++
+image::salesforce-mktg-3-dc-update.png[SMC-update]
++
+* *Upsert* operation:
++
+** *Object Type* - Type of API object to upsert
+** *Api Objects* - Array of one or more API objects
++
+image::salesforce-mktg-3-dc-upsert.png[SMC-upsert]
++
+* *Configure* operation:
++
+** *Configurations* - Array of one or more configurations to create
+** *Action* - Create, Update, or Delete
+** *Object Type* - Type of configuration
++
+image::salesforce-mktg-3-dc-configure.png[DC-configure]
++
+* *Perform* operation:
++
+** *Definitions* - Array of one or more definitions for the Perform operation
+** *Action* - Action to perform. Can have the value `Start`, `Stop`, or `Get`. `Max Count`
+** *Object Type* - Type of object to do a perform on
++
+image::salesforce-mktg-3-dc-perform.png[DC-perform]
++
+* *Schedule Start* operation:
++
+** *Object Type* - Type of object on which to have a schedule
+** *Interactions* - Array of one or more interactions in the schedule operation
++
+image::salesforce-mktg-3-dc-schedule-start.png[DC-schedule-start]
++
+* *Retrieve* operation:
++
+** *Query* - Query describing the objects to retrieve
++
+image::salesforce-mktg-3-dc-retrieve.png[DC-retrieve]
+
+== Add the Connector to a Studio Project
```

Comment:
Here, I would delete everything except the "Add the Connector in Studio" and then promote "Add the Connector in Studio" to a Level 2 heading.

Reply:
So this put the Configure the Connector in Studio lower under Add the Connector in Studio, which isn't right. I did a quick fix with the headers but this is definitely something we should put in the new template when we have time.